packages/flutter/test/cupertino/app_test.dart

```diff
@@ -481,6 +481,39 @@ void main() {
     },
   );
 
+  testWidgets('CupertinoApp has the correct default status bar icon brightness by default', (
+    WidgetTester tester,
+  ) async {
+    await tester.pumpWidget(const CupertinoApp(home: CupertinoPageScaffold(child: Text('Hello'))));
+
+    expect(
+      SystemChrome.latestStyle,
+      const SystemUiOverlayStyle(statusBarIconBrightness: Brightness.dark),
+    );
+  });
+
+  testWidgets(
+    'CupertinoApp has the correct default status bar icon brightness when the system is set to dark theme',
```

Comment:
```suggestion
    'CupertinoApp has the correct default status bar icon brightness when the system is set to dark',
```

Just a nit suggestion to avoid confusion with device brightness and Theme (from Flutter) brightness. :)

Reply:
done